ci-operator/jobs/devfile/registry/devfile-registry-main-periodics.yaml

```diff
@@ -10,20 +10,10 @@ periodics:
     org: devfile
     repo: registry
   labels:
-    ci-operator.openshift.io/variant: v4.12
+    ci-operator.openshift.io/variant: v4.13
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-devfile-registry-main-v4.12-rhtap-nightly-devfile-compatibility
-  reporter_config:
-    slack:
-      channel: '#team-has-github-alerts'
-      job_states_to_report:
-      - success
-      - failure
-      - error
-      report_template: '{{ if eq .Status.State "success" }} :success-kid: {{ else
-        }} :fuuuuuu: {{ end }} Job *{{.Spec.Job}}* ended with *{{.Status.State}}*.
-        <{{.Status.URL}}|View logs>'
```

Comment:
@thepetk This needs to be re-added to keep our slack notifications, job generation is currently removing this.

Reply:
@michael-valdron nice catch thanks :) I've re-added in the previous version of the PR but not the second time I ran the `make jobs`. I've re-added again!